pkg/appfile/template/manager.go

```diff
@@ -7,7 +7,7 @@ import (
 
 type Manager interface {
 	IsTrait(key string) bool
-	LoadTemplate(key string) string
+	LoadTemplate(key string) (defName, tmpl string)
```

Comment:
we don't need to change this function， instead, the key should be definitionname